cmd/handler-api.go

```diff
@@ -86,7 +86,7 @@ func availableMemory() (available uint64) {
 		return
 	}
 
-	available = memStats.Available / 2
+	available = memStats.Total / 2
```

Comment:
Available is not Free:
```
	// RAM available for programs to allocate
	//
	// This value is computed from the kernel specific values.
```

This seems like the correct value.

Reply:
Oh I see - then its my mistake @klauspost

Reply:
Not sure why this is divided by two here. This function is only returning the available memory, the remaining calculation happens outside